CHANGES.txt

```diff
@@ -5,11 +5,11 @@ Changes for crash
 Unreleased
 ==========
 
- - Expanded the list of keywords used by autocompletion and 
+ - Expanded the list of keywords used by autocompletion and
    autocapitlization.
 
- - Automatically capitalize keywords while typing,
-   e.g. ``select`` -> ``SELECT``
+ - Automatically capitalize keywords while typing, e.g. ``select`` -> ``SELECT``
+   **This feature is experimental and may be removed in future versions!**
```

Comment:
shouldn't we mention that it's off by default? otherwise it implies to me that it is enabled from now on